src/meta-schema.ts

```diff
@@ -36,6 +36,7 @@ const ONE_SCHEMA_META_SCHEMA: JSONSchema4 = {
           required: ['Name', 'Response'],
           properties: {
             Name: { type: 'string', pattern: '[a-zA-Z0-9]+' },
+            Description: { type: 'string' },
```

Comment:
Q for reviewers. Should we make `Description` required? I couldn't decide. Reasons why I like the idea of making it required:

- Forces at least _some_ documentation.
- OpenAPI does require a `description` field. Right now, when it's not provided, I'm setting `'None'`.

Reply:
IMO no, I foresee a lot of people being annoyed at having to add it and putting in junk.